app/results/routing.go

```
@@ -93,6 +93,11 @@ func routeGroup(r *http.Request, cfg config.Config) (int, http.Header, []byte, e
 	vars := mux.Vars(r)
 	tenantcfg, err := authentication.AuthenticateTenant(r.Header, cfg)
 	if err != nil {
+		if err.Error() == "Unauthorized" {
+			code = http.StatusUnauthorized
+			return code, h, output, err
```

Should we also return an error message in the response or just the error code? In such a case we have to also return the output var with an appropriate value.